.ci/build.sh

```diff
@@ -12,5 +12,11 @@ else
   python manage.py fetch_old_gci_task_data _site || true
 fi
 
+python manage.py import_organization_data
```

Comment:
create a single command to invoke all the other commands, so that the python runtime doesnt need to start so many times.